packages/design-system/src/styles/components/_Dropdown.scss

```diff
@@ -58,16 +66,48 @@
   .ds-c-field--inverse + & {
     color: var(--color-base);
   }
+
+  // When dropdown is open (not `hidden`), add border radius to bottom of menu
+  &:not([hidden]) {
+    // We don't have a good way to create alpha versions of colors using our current hex value tokens
+    // Sass has color functions, but they can't be used in CSS custom properties because CSS custom properties are evaluated at runtime
+    // Hex values can't be used in native CSS color functions like `hsla()` or `rgba()`
+    // Creating a temporary variable until we discuss a better solution
+    --shadow-color: 0deg 0% 50%;
+    animation: fade-in var(--animation-speed-1) var(--animation-ease-in-out-expo);
+    border-radius: 0 0 var(--dropdown-menu__border-radius--open)
+      var(--dropdown-menu__border-radius--open);
+    box-shadow: 0px 0.2px 0.2px hsl(var(--shadow-color) / 0.43),
```

Comment:
We don't have a drop-shadow token that matches the Sketch. I tried repurposing the same drop-shadow from dialog, but it was too dark/severe compared to the design. This was the closest I could get it.

Reply:
I think we should try to stick with the shadows we have in the design system rather than trying to match Sketch exactly. I can't imagine there was pixel-perfect intentionality with the drop-shadow design. Was there, @brentschneider?

Reply:
Like is there a way simpler version of this that would work? Could we just match the calendar picker drop shadow or the tooltip drop shadow?

Reply:
There's an issue using our date picker shadows.

The shadow spreads around the HTML element its attached to and I can't find a way to "turn off" one side of the shadow like you would with a border. So what we get is a shadow that encircles the entire element. It doesn't look bad on the date picker because the calendar opens a distance away from the field, but because the dropdown menu is attached to the field, a gray haze fills the field (see attached images to see this issue). 

The few ways I could think to fix this resulted in unwanted side effects:
1. I could place the dropdown menu behind the field (adjust the z-index), but this results in the field's focus ring appearing on top of the menu. 
2. I could adjust the offset-y of the shadow so it moves the shadow down far enough it doesn't overlap the field, but because the blur-radius is so large (17px) it makes the menu appear as if it's very far off the page. 
3. The previous way I had shadows _seemed_ to fix this overlap issue, but it results in more one-off code and might be harder to maintain in the long run (no corresponding shadow color token, no tokens for the many overlapping alphas, etc.).

![Screenshot 2023-06-06 at 2 41 22 PM](https://github.com/CMSgov/design-system/assets/5159392/09fabe1c-672f-44e1-8137-6492f14ed672)
![Screenshot 2023-06-06 at 2 41 30 PM](https://github.com/CMSgov/design-system/assets/5159392/8dbd391f-aeb7-4506-95c4-4eeda8dbeb62)

This is the way I had the shadows (the overly complex way). I changed the color of the shadow to red so you could more easily see if it was overlapping and I can't tell if it is:
![Screenshot 2023-06-06 at 2 56 19 PM](https://github.com/CMSgov/design-system/assets/5159392/e2d03fb6-9027-498c-b87d-24dcc0529e49)

Reply:
Would it be possible to place put the shadow on a pseudo-element child of the wrapper element of the whole component? Maybe the wrapper could have a z-index of 1 and the shadow element 0?

Reply:
I went with an off-menu option - used `clip-path` to cut the shadow off from overlapping the field. The only downside here is it results in:
- The shadow partially overlapping the focus ring
- A harsh line between the shadow and no-shadow (the light grey color on white background makes this less obvious imo)

I shared with Brent and this appears to be the exact same solution (and problems) he faced when designing in Sketch and got a 👍 on it.

![image](https://github.com/CMSgov/design-system/assets/5159392/fbc1439b-a19d-451a-bccf-49019b2fdd1a)